### ISSUE TITLE
Missing files of the @lwc/eslint-plugin-lwc module

### ISSUE THREAD

**Issue by berehovskyi** [salesforce/eslint-plugin-lwc#16]
Module installation

> npm install eslint babel-eslint @lwc/eslint-plugin-lwc --save-dev

or even

> npm install eslint babel-eslint @lwc/eslint-plugin-lwc@latest --save-dev

occurs with missing files, which are not included in the module:
```
eslint-plugin-lwc/lib/rules/consistent-component-name.js
eslint-plugin-lwc/lib/rules/no-dupe-class-members.js
```

and these files should have been mentioned in `eslint-plugin-lwc/lib/index.js` file.
As far as I see these two rules was added recently but the whole module wasn't updated.

**Comment by pmdartus** [comment]
The rules that you are referring to have been added after the release of `v0.6.0`.
`consistent-component-name` and `no-dupe-class-members` rules will be part of the next release.

**Comment by pmdartus** [comment]
@berehovskyi I just published the `0.7.0` version of the `@lwc/eslint-plugin-lwc` package.
